storage_service/locations/api/resources.py

```diff
@@ -592,6 +595,30 @@ def recover_aip_request(self, request, bundle, **kwargs):
         return http.HttpResponse(status=status_code, content=response_json,
             content_type='application/json')
 
+    @_custom_endpoint(expected_methods=['post'],
+        required_fields=('location_uuid', ))
+    def move_request(self, request, bundle, **kwargs):
+        """
+        Move package to another location
+        """
+        request_info = bundle.data
+        package = bundle.obj
+
+        if package.status == Package.MOVING:
+            message = "Move already in progress."
+            success = False
+        else:
```

Comment:
Do we want to move on every package status?  Probably not for Failed, Deleted, Pending (not yet on SS), maybe not for Delete requested or recovery request.